extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java

```diff
@@ -84,7 +85,7 @@ public void onFetchPage(@Nonnull final Downloader downloader) throws IOException
         final String[] channelInfo = channel_path.split("/");
         String id = "";
         // If the url is an URL which is not a /channel URL, we need to use the
-        // navigation/resolve_url endpoint of the youtubei API to get the channel id. Otherwise, we
+        // navigation/resolve_url endpoint of the internal API to get the channel id. Otherwise, we
```

Comment:
Why do you use internal API instead of youtubei API? pbj API is also an internal API.